client/my-sites/checkout/checkout/checkout-container.jsx

```diff
@@ -80,6 +84,16 @@ class CheckoutContainer extends React.Component {
 
 		return (
 			<>
+				{ this.props.isComingFromFrankenflow && (
+					<Button
+						borderless
+						className="navigation-link back"
+						onClick={ () => window.history.go( -2 ) } // going back to signup flow and skipping '/launch' step
+					>
+						<Gridicon icon="arrow-left" size={ 18 } />
+						{ this.props.translate( 'Back' ) }
+					</Button>
```

Comment:
@simison Do you think we need to make this more generic at the moment?

Reply:
I think it's fine as-is otherwise but I'd call things `isComingFromGutenboarding` since that's more clear to outsiders.